about.html

```diff
@@ -0,0 +1,221 @@
+              <img
+                src="assets/images/Global-ERG-Summit-Logo-2x.png"
+                width="60%"
+                alt=""
+              />
+            </div>
+    
+            <ul class="list-group">
+              <li class="list-group-item">
+                <a href="./index.html">Home</a>
+              </li>
+              <li class="list-group-item">
+                <a href="">Programs</a>
+              </li>
+              <li class="list-group-item">
+                <a href="">Join</a>
+              </li>
+              <li class="list-group-item">
+                <a href="">Sponsor</a>
+              </li>
+              <li class="list-group-item">
+                <a href="">News</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- Navigation -->
+        <nav class="navbar navbar-custom">
+          <div class="nav-container">
+            <button class="navbar-toggler" type="button">
+              <i class="fas fa-bars"></i>
+            </button>
+
+            <a class="navbar-brand">
+              <img src="assets/images/Global-ERG-Summit-Logo-2x.png" alt="" />
+            </a>
+            <div class="navbar-nav-container d-flex align-items-center">
+              <ul class="navbar-nav mr-auto mb-2 mb-lg-0">
+                <li class="nav-item">
+                  <a class="nav-link" href="./index.html"
+                    >Home</a
+                  >
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Program</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Join</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">News</a>
+                </li>
+              </ul>
+
+              <div class="header-action-btn">
+                <button type="button" class="btn btn-custom">
+                  CC Campaign
+                </button>
+              </div>
+            </div>
+          </div>
+        </nav>
+      </header>
+
+      <!-- Banner Section -->
+      <section id="banner-section">
+        <div class="banner-container">
+          <h4 class="display-4 sub-title">"Musical Concert"</h4>
+          <h1 class="display-1 main-title text-uppercase">
+            <span>We are</span> <br />
+            the song
+          </h1>
+          <p class="banner-description">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. A earum, fugiat sapiente iure aspernatur porro perspiciatis debitis ipsa vitae modi natus illum ea vel, voluptatum reprehenderit hic optio!
+          </p>
+
+          <div class="date-location-info">
+            <h3 class="font-weight-bold">Vibrant Music Group</h3>
+            <p>We live in Accra, Ghana</p>

```

Comment:
- [ ] Please centre the heading as seen in the original design. 👍 
- [ ] Please add a white border around the lorem ipsum paragraph as seen in the original design.
- [ ] Please add the contact email.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/190830011-fe6bf094-3cad-4cb4-8ed8-d45197db5ecd.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/190830122-65624f5d-b9a7-4f2a-b443-21069c6c1050.png)